host_files/asimov/nginx/12urenloop.be

```diff
@@ -0,0 +1,40 @@
+upstream 12urenloop-site {
+   server localhost:8080;
```

Comment:
Best niet op 8080 laten luisteren, dit is een van de poorten die extern openstaan.

Reply:
staat op 3001 nu